Szymon - saucedemo/src/test/java/zadanie5/InventoryTests.java

```diff
@@ -0,0 +1,139 @@
+        assertEquals("Total value is not correct for one backpack", "Total: $32.39", driver.findElement(By.xpath("//div[@class='summary_total_label']")).getText());
+
+        goToCheckoutCompleteStep5();
+        assertEquals("order is not complete or thank you text changed", "THANK YOU FOR YOUR ORDER", driver.findElement(By.xpath("//h2[@class='complete-header']")).getText());
+    }
+
+    @Test
+    public void buyFewAndVerifyContainerTest(){
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Backpack')]/../../following-sibling::div//button")).click();
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Bike Light')]/../../following-sibling::div//button")).click();
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Bolt T-Shirt')]/../../following-sibling::div//button")).click();
+        goToShoppingCartContainerStep1();
+        String cartList = driver.findElement(By.xpath("//div[@class='cart_list']")).getText();
+        assertThat(cartList).contains("Sauce Labs Backpack", "Sauce Labs Bike Light", "Sauce Labs Bolt T-Shirt");
+    }
+
+    @Test
+    public void buyFewAndVerifySumTest(){
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Backpack')]/../../following-sibling::div//button")).click();
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Bike Light')]/../../following-sibling::div//button")).click();
+        driver.findElement(By.xpath("//div[contains(text(),'Sauce Labs Bolt T-Shirt')]/../../following-sibling::div//button")).click();
+        goToShoppingCartContainerStep1();
+        goToCartCheckoutStep2();
+        fillYourInformationStep3();
+        goToOverviewStep4();
+        assertEquals("Total value is not correct for three items", "Total: $60.45", driver.findElement(By.xpath("//div[@class='summary_total_label']")).getText());
+    }
+
+
+
+    public static void sleepToSee(int seconds){
+        seconds = seconds*1000;
+        try {
+            Thread.sleep(seconds);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+    private static void login(){
+        WebElement usernameField = driver.findElement(By.xpath("//input[@type='text']"));
+        usernameField.sendKeys("standard_user");
+
+        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));
+        passwordField.sendKeys("secret_sauce");
+
+        WebElement loginButton = driver.findElement(By.xpath("//input[@type='submit']"));
+        loginButton.click();
+    }
+    private static void openHamburgerMenu() {
+        driver.findElement(By.xpath("//button[contains(text(),'Open Menu')]")).click();
+    }
+    private static void logout() {
+        openHamburgerMenu();
+        WebElement logoutSidebarLink = driver.findElement(By.id("logout_sidebar_link"));
+        new WebDriverWait(driver, 5)
+                .until(p -> logoutSidebarLink.getLocation().x > 0);
```

Comment:
Lambda -> mi nie zadzialala z pomem, ktory masz w projekcie.
Intellij mi to naprawil poprzez dodanie 

```

Reply:
Chodzi o to, ze programowanie funkcjonalne i lambdy zostaly wprowadzone w javie 8, a maven by default uzywa 7.